scripts/utils/dependencies-json.ts

```diff
@@ -1,8 +1,10 @@
-import { readdir, readFile, writeFile } from 'fs-extra';
+import fs from 'fs-extra';
```

Comment:
unfortunately with this setup we need to make this change since `fs` is a commonjs module, so attempting to import particular symbols from it gives this error:

<img width="967" alt="Screenshot 2023-02-13 at 11 54 11 AM" src="https://user-images.githubusercontent.com/6207644/218521178-3c6539e0-baeb-4c91-b21e-6bd87669db00.png">

doing

```ts
import fs from 'fs-extra';

const { readdir, readFile, writeFile } = fs;
```

works fine, however.